app/main.py

```diff
@@ -1,8 +1,31 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
```

Comment:
The `people` dictionary is used to store instances of `Person` by name. This approach assumes that all names are unique, which might not be the case. Consider handling potential name conflicts or using a different identifier.